### PR TITLE
Business Logic & API: Document Endpoint Get and PUT PNG/JPEG

### PR DESCRIPTION
**Pull request by lavanya-bmw**

### CODE REVIEW

File: cx-portal/src/components/pages/AppReleaseProcess/components/AppMarketCard/index.tsx [hunk 1]
```diff
@@ -199,22 +201,31 @@ export default function AppMarketCard() {
...
-  const [salesManagerId, setSalesManagerId] = useState(null)
+  const [salesManagerId, setSalesManagerId] = useState<string | null>(null)
+  const [fetchDocumentById] = useFetchDocumentByIdMutation()
+  const [cardImage, setCardImage] = useState('')
+  const fetchAppStatus = useFetchAppStatusQuery(appId ?? '').data
+
+  useEffect(() => {
+    dispatch(setAppStatus(fetchAppStatus))
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [fetchAppStatus])
```
Comment (oyo) [inline cx-portal/src/components/pages/AppReleaseProcess/components/AppMarketCard/index.tsx:221]: what's the reason why this eslint rule got disabled?

File: cx-portal/src/components/pages/AppReleaseProcess/components/ValidateAndPublish/index.tsx [hunk 2]
```diff
@@ -154,11 +180,11 @@ export default function ValidateAndPublish({
                   ? appStatusData?.descriptions?.filter(
                       (lang: { languageCode: string }) =>
                         lang.languageCode === 'en'
-                    )[0].shortDescription
+                    )[0]?.shortDescription
                   : appStatusData?.descriptions?.filter(
                       (lang: { languageCode: string }) =>
                         lang.languageCode === 'de'
-                    )[0].shortDescription
+                    )[0]?.shortDescription
```
Comment (oyo) [inline cx-portal/src/components/pages/AppReleaseProcess/components/ValidateAndPublish/index.tsx:187]: can you please change this entire logic to:
`description={`
`  appStatusData?.descriptions?.filter(`
`    (lang: { languageCode: string }) => lang.languageCode === i18next.language`
`)}`

**Review by oyo — CHANGES_REQUESTED** [review]

**Comment by jjeroch** [comment]
cleaned up the changelog

**Comment by jjeroch** [comment]
2 comments left from martin

**Review by oyo — APPROVED** [review]
looks good now

**Pull request merged**